examples/with-eip-1193-provider/src/components/turnkey-provider-context.tsx

```diff
@@ -0,0 +1,77 @@
+"use client";
+
+import React, { createContext, useContext, useEffect, useState } from "react";
+import {
+  TurnkeyEIP1193Provider,
+  createEIP1193Provider,
+} from "@turnkey/eip-1193-provider";
+import { WebauthnStamper } from "@turnkey/webauthn-stamper";
+import { TurnkeyClient } from "@turnkey/http";
+import { numberToHex } from "viem";
+import { sepolia } from "viem/chains";
+
+type TurnkeyProviderContextType = {
+  turnkeyProvider: TurnkeyEIP1193Provider | null;
+  setTurnkeyProvider: React.Dispatch<
+    React.SetStateAction<TurnkeyEIP1193Provider | null>
+  >;
+};
+
+const TurnkeyProviderContext = createContext<
+  TurnkeyProviderContextType | undefined
+>(undefined);
+
+export const TurnkeyProviderProvider: React.FC<{
+  children: React.ReactNode;
+}> = ({ children }) => {
+  const [turnkeyProvider, setTurnkeyProvider] =
+    useState<TurnkeyEIP1193Provider | null>(null);
+
+  useEffect(() => {
+    const initializeProvider = async () => {
+      const storedData = localStorage.getItem("turnkeyProviderData");
```

Comment:
tiny nit: would be nice to have consts like `"turnkeyProviderData"` declared separately